docs/ops/movement/Unique_10.md

```diff
@@ -0,0 +1,119 @@
+# Unique  {#openvino_docs_ops_comparison_Unique_10}
+
+**Versioned name**: *Unique-10*
+
+**Category**: *Data movement*
+
+**Short description**: *Unique* finds unique elements of a given input tensor. Depending on the operator's attributes it either looks for unique values globally in the whole (flattened) tensor or performs the search along the specified axis.
+
+**Detailed description**
+The operator can either work in elementwise mode searching for unique values in the whole tensor or it can consider slices of the input tensor along the specified axis. This way the op is able to find unique subtensors in the input data. Except for the unique elements the operator also produces the indices of the unique elements in the input tensor and the number of occurences of unique elements in the input data.
+
+**Attributes**:
+
+* *axis*
+
+  * **Description**: *axis* specifies the input tensor's axis along which the subtensors are processed. When the axis is set to the maximum value of the underlying integer type it means that the input tensor should be processed elementwise (the operator then looks for unique individual numbers in the whole flattened tensor).
+  * **Range of values**: The accepted range of values is `[-R, R-1]` where `R` is the rank of the input tensor.
+  * **Type**: int
+  * **Default value**: MAX_INT
+  * **Required**: *no*
```

Comment:
Why do we have this attribute if we have dedicated input for this?

Reply:
This an oversight, thanks for catching it